log4j-samples/log4j-samples-loggerProperties/src/main/java/org/apache/logging/log4j/lookup/MapMessageLookup.java

```diff
@@ -52,7 +53,7 @@
     @Override
     public String lookup(final LogEvent event, final String key) {
         final Message msg = event.getMessage();
-        if (msg instanceof StringMapMessage) {
+        if (msg instanceof MapMessage) {
```

Comment:
Appending `Object` values to the stringbuilder on 69 is potentially dangerous, `MapMessage.get(String)` works around this by taking advantage of `ParameterFormatter.deepToString`.

Reply:
added